docs/source/user_guide/automl.ipynb

```diff
@@ -396,6 +396,59 @@
     "automl.rankings"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Recommendation Score\n",
+    "\n",
+    "If you would like a more robust evaluation of the performance of your models, you can look at the recommendation score for a model. The recommendation score is a weighted average of a number of default objectives for your problem type, normalized and scaled so that the final score can be interpreted as a percentage from 0 to 100."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl.get_recommendation_scores(use_pipeline_names=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl.get_recommendation_scores(priority=\"F1\", use_pipeline_names=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you would like to automatically rank your pipelines by this recommendation score, you can set `use_recommendation=True` when initializing `AutoMLSearch`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_recommendation = evalml.automl.AutoMLSearch(\n",
+    "    X_train=X_train,\n",
+    "    y_train=y_train,\n",
+    "    X_holdout=X_holdout,\n",
+    "    y_holdout=y_holdout,\n",
+    "    problem_type=\"binary\",\n",
+    "    use_recommendation=True,\n",
+    ")\n",
+    "automl_recommendation.search(interactive_plot=False)\n",
+    "\n",
+    "automl_recommendation.rankings"
```

Comment:
Spacing is off [here](https://feature-labs-inc-evalml--4156.com.readthedocs.build/en/4156/user_guide/automl.html#Recommendation-Score) for some reason
![Screenshot 2023-04-25 at 11 14 53 AM](https://user-images.githubusercontent.com/15271480/234323334-cf53f7d3-3660-4f6f-a6ed-d1147e4d8715.png)

Reply:
This is a weird one. It doesn't happen when running the jupyter notebooks directly, and it looks like the same issue comes up with other rankings dataframes as well - this one looks the worst since it has the most columns. I think I'll drop some unnecessary columns so that they aren't so squished, but I'm also open to other suggestions, as Google was unhelpful here.